tests/storage/hotplug.go

```diff
@@ -2193,3 +2193,11 @@ func verifyVolumeAndDiskVMIAdded(virtClient kubecli.KubevirtClient, vmi *v1.Virt
 		return nil
 	}, 90*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
 }
+
+func renameImgFile(pvc *corev1.PersistentVolumeClaim, newName string) {
```

Comment:
maybe we can add some error handling?

Reply:
Please note nothing require error handling.

`tests.RunPodAndExpectCompletion(pod)` return no error be cause it assert error by itsef, which we plan to convert it to return error.